activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1627,10 +1625,6 @@ def add_timestamps_for_alter(table_name, **options)
           ]
         end
 
-        def remove_timestamps_for_alter(table_name, **options)
-          remove_columns_for_alter(table_name, :updated_at, :created_at)
-        end
```

Comment:
This method is still necessary, although evidently we're missing a test to demonstrate why.

Given this migration:

```ruby
class RemoveTitleAndTimestampsFromPosts < ActiveRecord::Migration[7.0]
  def change
    change_table :posts, bulk: true do |t|
      t.remove :title, type: :string
      t.remove_timestamps
    end
  end
end
```

Before this change, the operations are combined into a single statement:

```sql
ALTER TABLE `posts` DROP COLUMN `title`, DROP COLUMN `updated_at`, DROP COLUMN `created_at`
```

Afterwards, that's no longer the case:

```sql
ALTER TABLE `posts` DROP COLUMN `title`
ALTER TABLE `posts` DROP COLUMN `updated_at`, DROP COLUMN `created_at`
```

Reply:
I added the missing test coverage in https://github.com/rails/rails/commit/217f0e81d1af895fdef2305200824805f23f2a74.

Reply:
Thanks for catching and testing this!

Restored this method in the latest revision.